funfact/loss.py

```diff
@@ -101,6 +134,13 @@ def _loss(self, model, target):
         return ab.multiply(target, ab.log(ab.divide(target, model)))
 
 
+class GlobalPhaseMSE(MatrixLoss):
```

Comment:
Could we rename it to something like `PhaseInvariantMSE` to make it more intuitive? 😄